.github/ISSUE_TEMPLATE/gc_member_onboarding.md

```diff
@@ -0,0 +1,192 @@
+2. Have read the [OpenTelemetry Governance Committee Charter](https://github.com/open-telemetry/community/blob/main/governance-charter.md)
+   as per requirements to stand for election.
+3. Have an email address, Google account, and GitHub account and is willing
+   to share those details with other GC members.
+   1. **Note**: Google account is mainly needed for Google Docs, but one may
+      prefer to use work email or personal email for calendar invites.
+
+
+## Access Grants
+This section ensures that the new member has access to all systems and has the
+necessary privileges to perform their role as a member of the GC.
+
+### 1Password
+1Password contains credentials for accounts managed by the GC as a group (i.e.
+not personal items).
+
+The existing member must:
+
+- [ ] Invite new member to [OpenTelemetry 1Password](https://opentelemetry.1password.com)
+      via email.
+- [ ] After invite is accepted, add them to the `Owners` group as a `Manager`.
+
+The  current member must:
+
+- [ ] Accept invitation and create new account on [OpenTelemetry 1Password](https://opentelemetry.1password.com).
+- [ ] Validate they have `Manager` access to `Owners` group, and that the group
+      contains only current GC members.
+- [ ] Validate they have `Full Access` to the `OpenTelemetry Governance` vault.
+      This should be given automatically via the `Owners` group.
+
+### CNCF Slack
+The main async medium for GC and TC members to communicate is a set of private
+channels on Slack.
+
+The existing member must:
+
+- [ ] Add new member to the following private channels:
+  - [opentelemetry-gc](https://cloud-native.slack.com/archives/C01S673T1NE)
+  - [opentelemetry-gc-tc](https://cloud-native.slack.com/archives/C02KF2PPUV9)
+  - [opentelemetry-gc-alum](https://cloud-native.slack.com/archives/C02KBUGC685)
+
+The new member must:
+
+- [ ] Create a [CNCF Slack](https://slack.cncf.io/) account if they do not
+  already have one.
+
+### CNCF Service Desk
+Interactions between the OpenTelemetry GC and CNCF are mostly driven via Service
+Desk tickets. Members can create tickets to discuss access to systems, licensing,
+funds, marketing, press, etc.
+
+The existing member must:
+
+- [ ] Create a Service Desk ticket to invite new member via email and add it to
+  OpenTelemetry group.
+
+The new member must:
+
+- [ ] Accept invitation to CNCF Service Desk and create an account.
+- [ ] Validate they have access to the [CNCF Service Desk](https://cncfservicedesk.atlassian.net/servicedesk/customer/portals)
+  (see more information about Service Desk in the [docs](https://github.com/cncf/servicedesk#readme)).
+- [ ] Validate they have access to [OpenTelemetry Requests](https://cncfservicedesk.atlassian.net/servicedesk/customer/user/requests?page=1&reporter=org-31).
+
+### GitHub
+Most of the management, enablement, support and documentation work done by GC
+members happens on GitHub.
+
+The existing member must:
```

Comment:
This will most likely change soon after https://github.com/open-telemetry/community/pull/1793, in which case the new member should just be able to open the necessary pull requests.

Reply:
I'm leaving it as-is for now, and will update later